src/app/components/Curation/Subhead/index.tsx

```diff
@@ -1,18 +1,20 @@
-import React, { Fragment, PropsWithChildren, useContext } from 'react';
+/** @jsx jsx */
```

Comment:
I'll need to check again why we do it, but we may need `/** @jsxRuntime classic */` here too for Next.JS compatibility. Not that we'll be using this component in the Next app soon, but will do in the future once we move other page types over to it.

There is a compilation reason why we use that specific jsxRuntime, will need to dig out the reason why!

Reply:
I keeeeeeep forgetting to do this 🤦 Will add now.

Reply:
Is it maybe this? https://stackoverflow.com/questions/66965774/emotion-css-prop-and-nextjs-new-jsx-runtime-error-pragma-and-pragmafrag-canno

Reply:
Yea I think its related to that and possibly Opera Mini not being compatible with the new runtime introduced in React 17.

Reply:
@amoore108 from what i've read the classic runtime uses react's legacy JSX transform and requires the `/** @jsxRuntime classic */` pragma at the top of the file to tell Babel to use the legacy transform. It's required for compatibility with older versions of React and certain third-party libraries that have not yet migrated to the new JSX transform.

Might it be possible to add something like this to the `Next` config file temporarily instead of adding it to the top of each new component?

`
module.exports = {
  reactJsxRuntime: 'classic'
}`


Or using `@babel/plugin-transform-react-jsx` to transform the JSX in the rest of the app that needs to use the Classic runtime and modifying: https://github.com/bbc/simorgh/blob/c4b1c4ff55ec12b645c4312f67bfbada5563467f/.babelrc.js

Reply:
Looking at the Git history for these jsxRuntime frags, they were first introduced with the Next.JS integration, so its likely we can isolate it to that app at least and try out using `preset-react` to do this for us.

Reply:
I'll spin up a PR and see what it does. There is no traffic to Opera Mini on the Next.JS page, so its worth testing now.